bin/crew

```diff
@@ -1202,54 +1157,41 @@ def archive_package(pwd)
   system "sha256sum #{pwd}/#{pkg_name} > #{pwd}/#{pkg_name}.sha256"
 end
 
-def remove(pkgName)
-
-  #make sure the package is actually installed
-  unless @device[:installed_packages].any? { |pkg| pkg[:name] == pkgName } || File.exist?("#{CREW_META_PATH}#{pkgName}.filelist")
-    puts "Package #{pkgName} isn't installed.".lightred
-    return
+def remove (pkgName)
+  # make sure the package is actually installed
+  unless installed?(pkgName, mode: 'json') or installed?(pkgName, mode: 'filelist')
+    return puts "Package #{pkgName} isn't installed.".lightred
   end
 
-  #if the filelist exists, remove the files and directories installed by the package
-  if File.file?("#{CREW_META_PATH}#{pkgName}.filelist")
-    Dir.chdir CREW_CONFIG_PATH do
-
-      #remove all files installed by the package
-      File.open("meta/#{pkgName}.filelist").each_line do |line|
-        begin
-          puts 'Removing file ' + line.chomp + ''.lightred if @opt_verbose
-          File.unlink line.chomp
-        rescue => exception #swallow exception
-        end
-      end
-
-      #remove all directories installed by the package
-      File.readlines("meta/#{pkgName}.directorylist").reverse.each do |line|
-        begin
-          puts 'Removing directory ' + line.chomp + ''.lightred if @opt_verbose
-          Dir.rmdir line.chomp
-        rescue => exception #swallow exception
-        end
-      end
-
-      #remove the file and directory list
-      File.unlink "meta/#{pkgName}.filelist"
-      File.unlink "meta/#{pkgName}.directorylist"
+  # remove the files and directories installed by the package
+  Dir.chdir CREW_CONFIG_PATH do
+    # remove all files installed by the package
+    puts "Removing files".lightred if @opt_verbose
+    FileUtils.rm_f \
+      File.readlines("meta/#{pkgName}.filelist", chomp: true),
+      verbose: @fileutils_verbose
 
+    # remove all directories installed by the package
+    puts "Removing directories".lightred if @opt_verbose
+    begin
+      FileUtils.rmdir \
+        File.readlines("meta/#{pkgName}.directorylist", chomp: true),
+        verbose: @fileutils_verbose
+    rescue Errno::EROFS # Read-only file system @ dir_s_rmdir
     end
```

Comment:
How is this better?  You got rid of all the verbosity so there is no way to see all the files and directories that are removed.